client/.eslintrc.json

```diff
@@ -20,6 +21,7 @@
   },
   "plugins": ["react"],
   "rules": {
```

Comment:
Use these rules 
`"rules": {
        "no-multiple-empty-lines": [
          "error",
          {
            "max": 1
          }
        ],
        "indent": [
            "warn",
            2,
            {"SwitchCase": 1}
        ],
        "linebreak-style": [
            "error",
            "unix"
        ],
        "quotes": [
            "error",
            "single"
        ],
        "semi": [
            "error",
            "always"
        ],
        "react/jsx-uses-vars": "error",
        "react/jsx-uses-react": "error",
        "react-hooks/rules-of-hooks": "error"
    }`

Reply:
Fixed 
Then i will changed warn to error